packages/split-view/README.md

```diff
@@ -120,9 +167,11 @@ import { SplitView } from '@spectrum-web-components/split-view';
 </sp-split-view>
 ```
 
-### Vertical Resizable & Collapsible
+</sp-tab-panel>
+<sp-tab value="collapsible">Resizable & collapsible</sp-tab>
+<sp-tab-panel value="collapsible">
 
-```html
+```html demo
 <sp-split-view
```

Comment:
I don't see the collapsible property being put on here (same for horizontal)... how does this collapsible feature work? 🤯 

Is it just automatically collapsible if there's no min? Could be useful to mention.